test/verify/check-machines

```diff
@@ -1396,6 +1403,28 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         self.waitVmRow(name, 'system', False)
         b.wait_not_present('#vm-{0}-state button:contains("view more")'.format(name))
 
+        # Try to delete VM which has a disk defined with a file as a source instead of a storage volume
+        # In this case, the libvirt's storage volume deletion API would fall but cockpit would fall back to deleting the file directly
+        name = "VM-with-file-disk"
+        disk = "/tmp/fakeDisk"
+        m.execute("touch {0}".format(disk))
+        args = self.createVm(name)
+        m.execute("virsh destroy {0}".format(name))
+        m.execute("virsh attach-disk --domain {0} --source {1} --target vdb --persistent".format(name, disk))
+
+        b.reload()
+        self.browser.enter_page('/machines')
+        self.goToMainPage()
+        b.click("#vm-{0}-action-kebab button".format(name))
+        b.click("#vm-{0}-delete".format(name))
+
+        b.wait_visible("#vm-{0}-delete-modal-dialog".format(name))
+        self.assertTrue(lambda: checkFilePresent(disk))
+        b.click("#vm-{0}-delete-modal-dialog button:contains(Delete)".format(name))
+        b.wait_not_present("#vm-{0}-delete-modal-dialog".format(name))
+        self.assertTrue(lambda: not checkFilePresent(disk))
```

Comment:
This is a bit weird -- please use `assertFalse` to check for negative conditions.  But moreover, `m.execute("test ! -e " + disk`